src/emulator/functionsEmulator.ts

```diff
@@ -426,7 +417,13 @@ export class FunctionsEmulator implements EmulatorInstance {
         return true;
       }
 
-      return this.getTriggerDefinitionByName(definition.name) === undefined;
+      // We want to add a trigger if we don't already have an enabled trigger
+      // with the same entryPoint.
+      const matchingTriggers = Object.values(this.triggers).filter((record) => {
+        return record.def.entryPoint === definition.entryPoint;
+      });
+      const anyEnabledMatch = matchingTriggers.some((def) => def.enabled);
```

Comment:
nit: why don't we just use one single call to `.some()` instead of `filter` + `some`?

Reply:
👍